app/assets/stylesheets/header.scss

```diff
@@ -2,12 +2,13 @@
 
 header {
   .navbar-brand {
-    img { width: 330px; }
+    img { width: 250px; }
   }
 
   .navbar {
-    padding: 40px 0 60px 0;
+    padding: 20px 0 20px 0;
```

Comment:
This is a bad idea. Navabar is present on every site, and banner only on homepage. Larger bottom padding makes it work on every single page. Please bring this back, in smaller version:
padding: 25px 0 45px 0;